### PR TITLE
Add Gromit-MPX

### PR DESCRIPTION
**Pull request by bk138**
Gromit-MPX is an on-screen annotation tool that works with any Unix
desktop environment under X11 as well as Wayland.

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build net.christianbeier.Gromit-MPX

File: net.christianbeier.Gromit-MPX.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+id: net.christianbeier.Gromit-MPX
+runtime: org.gnome.Platform # using GNOME for the time being as we need dconf for setting hotkeys under Wayland
+runtime-version: '3.36'
```
Comment (LeandroStanger) [inline net.christianbeier.Gromit-MPX.yml:3]: Update runtime to 3.38

Comment (bk138) [reply net.christianbeier.Gromit-MPX.yml]: ACK. See 1afcbc41f20f84a3a361708a44888b080d3cb2b0

**Comment by lu9dce** [comment]
bot, build net.christianbeier.Gromit-MPX

File: net.christianbeier.Gromit-MPX.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+id: net.christianbeier.Gromit-MPX
+runtime: org.gnome.Platform # using GNOME for the time being as we need dconf for setting hotkeys under Wayland
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: gromit-mpx
+finish-args:
+  - --socket=x11
```
Comment (barthalion) [inline net.christianbeier.Gromit-MPX.yml:7]: I guess it also works on Wayland natively as it's GTK3?

Comment (bk138) [reply net.christianbeier.Gromit-MPX.yml:7]: Nope, it depends on XWayland as of now.

**Pull request closed without merging**